### PR TITLE
Moving acs commands from vm to acs module

### PR DESCRIPTION
**Pull request by vishrutshah**
Based on my conversation with @brendandburns, I started from scratch (as #1497 has too many conflicts) to move commands out from vm those were not related to acs. 

Task 1 of work Item https://github.com/Azure/azure-cli/issues/1329

### CODE REVIEW

**Comment by vishrutshah** [comment]
@brendandburns I did start from scratch and moved the respective commands out from vm into acs. Please take a look if you get a chance. 

@yugangw-msft @derekbekoe @tjprescott @troydai  Please review this as you get a chance. Thanks!

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -690,3 +691,20 @@ def _mkdir_p(path):
...
+    client = cf_acs(None)
+    instance = client.get(resource_group_name, container_service_name)
+    instance.agent_pool_profiles[0].count = new_agent_count  # pylint: disable=no-member
+    return client.create_or_update(resource_group_name, container_service_name, instance)
+
+
+def list_container_services(resource_group_name=None):
+    ''' List Container Services. '''
+    from azure.cli.command_modules.acs.commands import cf_acs
+    client = cf_acs(None)
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: In commands.py, add `cf_acs` to the command registration then the first argument to this method can be `client`.
This was the case when these commands were in the vm module.
https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py#L176

The same can be done to `update_acs` above.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py [hunk 2]
```diff
@@ -7,6 +7,24 @@
 
 from azure.cli.core.commands import cli_command
 
+
+def cf_acs(_):
```
Comment (vishrutshah) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py]: Yes definitely, I've created `_client_factory.py` w.r.t PR #2110 I'll add this method into it once that PR gets merged and update this PR.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 1 above)
Comment (vishrutshah) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: Sure, will update it.

**Comment by vishrutshah** [comment]
@troydai No dependencies has changed but only packages https://github.com/Azure/azure-cli/pull/2098/files#diff-d1135a66adf72a654e5b66b1bc52378eR57 needed to be changed.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 1 above)
Comment (vishrutshah) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done :)

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py (hunk 2 above)
Comment (vishrutshah) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py]: updated

**Pull request merged**